### PR TITLE
RxSwift 5.0.0 support

### PR DESCRIPTION
**Pull request by rynecheow**
## Description
* Supports RxSwift 5.0.0 by modifying extensions to use `Observable.Element` instead of `Observable.E`

## Checklist

- [x] this PR is based on develop or a 'develop related' branch
- [x] the commits inside this PR have explicit commit messages
- [x] the Jazzy documentation has been generated (if needed -> Jazzy RxFlow)

### CODE REVIEW

File: RxFlowTests/RxFlowTests.swift [hunk 1]
```diff
@@ -0,0 +1,33 @@
+//
```
Comment (twittemb) [inline RxFlowTests/RxFlowTests.swift]: Is there any reason to have this file in your commit ?

Comment (rynecheow) [reply RxFlowTests/RxFlowTests.swift]: Was totally an accident! Sorry about that. Removed in a following commit.

**Comment by rynecheow** [comment]
Hey @twittemb, would you like to give it another shot?

**Comment by twittemb** [comment]
Thanks for your contribution 👍

**Comment by rynecheow** [comment]
@twittemb Do you have any expected release/tag date of this PR?

**Comment by rynecheow** [comment]
@twittemb Do you have any expected release/tag date of this PR?

**Comment by twittemb** [comment]
Hi @rynecheow 

I will try to release a 2.3.0 asap (today or tomorrow).

thanks.

**Pull request merged**